galaxy/templates/deployment-job.yaml

```diff
@@ -58,8 +58,13 @@ spec:
 {{- end }}
             - name: GALAXY_CONFIG_DATABASE_CONNECTION
               value: postgresql://{{ .Values.postgresql.galaxyDatabaseUser }}:$(GALAXY_DB_USER_PASSWORD)@{{ template "galaxy-postgresql.fullname" . }}/galaxy
-          command: ["python", "/galaxy/server/scripts/galaxy-main"]
-          args: ["-c", "/galaxy/server/config/galaxy.yml", "--server-name", "$(POD_NAME)", "--attach-to-pool", "job-handlers"]
+          command: ["/bin/bash", "-c", "python /galaxy/server/scripts/galaxy-main -c /galaxy/server/config/galaxy.yml --server-name $(POD_NAME) --attach-to-pool job-handlers 2>&1 | tee /tmp/log"]
+          readinessProbe:
+            exec:
+              command:
+                - "/bin/bash"
+                - "-c"
+                - "tail -n 10 /tmp/log | grep \"is running\""
```

Comment:
Tailing only 10 lines could miss the "is running" line if jobs are picked up between when it starts running and the end of a period to re-check:
eg: if a user has 11 jobs queued in the database, the check fails 1 sec before the running line, and by the time it checks again, the handler picked up 11 jobs (it might actually be just 6 jobs cause I think it logs a line when the job is picked up and another when it's dispatched to a runner, but regardless X # of jobs that would log 11+ lines). Very edge case and will most likely not happen, but it seems like an unnecessary risk.
Perhaps grep the file directly? Given that it's just a readiness probe and it's running only once at the beginning, the file should never be incredibly big, so it should be fine.